lib/board.rb

```diff
@@ -13,4 +13,15 @@ def get_space(row, column)
   def mark_space(token, space)
     values[space - 1] = token
   end
+
+  def full?
+    open_board = (1..9).to_a
+    open_spaces = values & open_board
+    open_spaces.empty?
+  end
+
+  def available?(input)
+    space = values[Integer(input) - 1]
```

Comment:
The move looks good. One other suggestion I'd make here: since we have the separate validator for numbers, I'd say this method should just take the number of the space to check rather than having to convert it here and then again [here](https://github.com/karen-olson/tic-tac-toe-basic/pull/4/files#diff-d055b7ab2b905af3de1536ecccf2be01e99cfaedf43a7c8ae2bca93263cea5efR32). Then the Board only has to worry about numeric spaces as part of its API.